src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_panel.tsx

```diff
@@ -135,31 +136,42 @@ export const WorkspacePermissionSettingPanel = ({
 
   return (
     <>
-      <EuiFlexGroup alignItems="center" gutterSize="s">
-        <EuiFlexItem style={{ maxWidth: 150 }}>
-          <EuiFormLabel id={PERMISSION_TYPE_LABEL_ID}>
-            {i18n.translate('workspaceForm.permissionSetting.typeLabel', {
-              defaultMessage: 'Type',
-            })}
-          </EuiFormLabel>
-        </EuiFlexItem>
-        <EuiFlexItem style={{ maxWidth: 400 }}>
-          <EuiFormLabel id={PERMISSION_COLLABORATOR_LABEL_ID}>
-            {i18n.translate('workspaceForm.permissionSetting.collaboratorLabel', {
-              defaultMessage: 'Collaborator',
-            })}
-          </EuiFormLabel>
-        </EuiFlexItem>
-        <EuiFlexItem style={{ maxWidth: 150 }}>
-          <EuiFormLabel id={PERMISSION_ACCESS_LEVEL_LABEL_ID}>
-            {i18n.translate('workspaceForm.permissionSetting.accessLevelLabel', {
-              defaultMessage: 'Access level',
-            })}
-          </EuiFormLabel>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false} style={{ width: 40 }} />
-      </EuiFlexGroup>
-      <EuiSpacer size="xs" />
+      {permissionSettings.length > 0 && (
+        <>
+          <EuiFlexGroup alignItems="center" gutterSize="s">
+            <EuiFlexItem style={{ maxWidth: 150 }}>
+              <EuiFormLabel id={PERMISSION_TYPE_LABEL_ID}>
+                {i18n.translate('workspaceForm.permissionSetting.typeLabel', {
+                  defaultMessage: 'Type',
+                })}
+              </EuiFormLabel>
+            </EuiFlexItem>
+            <EuiFlexItem style={{ maxWidth: 400 }}>
+              <EuiFormLabel id={PERMISSION_COLLABORATOR_LABEL_ID}>
+                {i18n.translate('workspaceForm.permissionSetting.collaboratorLabel', {
+                  defaultMessage: 'Collaborator',
+                })}
+              </EuiFormLabel>
+            </EuiFlexItem>
+            <EuiFlexItem style={{ maxWidth: 150 }}>
+              <EuiFormLabel id={PERMISSION_ACCESS_LEVEL_LABEL_ID}>
+                {i18n.translate('workspaceForm.permissionSetting.accessLevelLabel', {
+                  defaultMessage: 'Access level',
+                })}
+              </EuiFormLabel>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiButtonIcon
+                color="text"
+                iconType="trash"
+                size="xs"
+                style={{ visibility: 'hidden' }}
+              />
+            </EuiFlexItem>
```

Comment:
I am a little bit confused on this element, it contains a hidden element. Could you please elaborate more here?

Reply:
I think this is a placeholder to align column label with inputs. I think we can change to fixed width div.

Reply:
I c, thanks. A comment here would be preferred.

Reply:
Sure. Will add some comments above it.